src/config.h

```diff
@@ -192,7 +192,8 @@ struct Config {
         std::unordered_map<std::string, std::string> output_names_forwarder;
         bool run_on_prompt{true};
         bool run_on_token_gen{true};
-        int32_t reset_session_idx{-1};
+        int reset_session_idx{-1};  // Some models cannot keeo all the ort sessions in memory at once due to memory constraints.
```

Comment:
Small typo: keeo -> keep

Should you put a comment saying this is a temporary workaround and will be ignored once the proper fix in QNN is in?